connectors/tests/test_sync.py

```diff
@@ -700,5 +730,3 @@ async def test_concurrent_syncs(mock_responses, patch_logger, set_env):
 
     # make sure we synced the three connectors
     patch_logger.assert_present("[1] Sync done: 1 indexed, 0  deleted. (0 seconds)")
-    patch_logger.assert_present("[2] Sync done: 1 indexed, 0  deleted. (0 seconds)")
```

Comment:
why do you remove this coverage? we want to make sure we sync the 3  connectors

Reply:
Because I will soon rewrite this test when I refactor `SyncService`, where I will mock `ConcurrentTasks`, and 
```python
calls = [
    call(functools.partial(self._one_sync, connector1, es)),
    call(functools.partial(self._one_sync, connector2, es)),
    functools.partial(self._one_sync, connector3, es),
]
syncs.put.assert_has_awaits(calls)
```

I will fix the test in this PR.

Reply:
ah ok, great thanks! let's make sure we don't lose that coverage

Reply:
Done here: https://github.com/elastic/connectors-python/pull/460/commits/73454007a23342b0b04346d5f2230da9351b6942